.config/hakari.toml

```diff
@@ -10,6 +10,7 @@ dep-format-version = "2"
 # Hakari works much better with the new feature resolver.
 # For more about the new feature resolver, see:
 # https://blog.rust-lang.org/2021/03/25/Rust-1.51.0.html#cargos-new-feature-resolver
+# Have to keep the resolver still here, despite it's now the default for 2021 edition.
```

Comment:
Why do we need to keep this if it's the default?

Reply:
Hakari requires this field to be present, so there has to be something:

```
Work/neon/neon kb/neon-utils:origin/kb/neon-utils ❯ cargo hakari generate          
Error: 
   0: error deserializing Hakari config at /Users/someonetoignore/Work/neon/neon/.config/hakari.toml
   1: missing field `resolver` at line 1 column 1
```

Reply:
I think It is the default in cargo, not in hakari

Reply:
Phrased this as 

```
# Have to keep the resolver still here since hakari requires this field,
# despite it's now the default for 2021 edition & cargo.
```